constraint/deploy/crds.yaml

```diff
@@ -137,6 +139,8 @@ spec:
                             type: array
                         type: object
                       validation:
+                        default:
+                          legacySchema: true
                         properties:
                           legacySchema:
                             default: true
```

Comment:
How do these interact?  Do we need both?

Reply:
I see now that it serves two cases: `Validation: nil` and `Validation: &Validation{}`